instance_skel.js

```diff
@@ -177,12 +177,39 @@ instance.prototype.addUpgradeScript = function () {
 	)
 }
 
+instance.prototype.serializeIsVisibleFn = function (options = []) {
```

Comment:
I would prefer this method to not be on the class, especially as it doesnt need anything from the class.
As revealed in a slack thread, some module devs look at this file to see what methods are available, and even though this is nothing they will ever need to call, someone will ask about it.

Also could this be dangerous if a module dev decides to override it? Probably not, no worse than the lack of data validation elsewhere

If you want to keep it here (I know @krocheck prefers having methods on classes rather than loose functions), then it should at least have a comment calling it an internal method

Reply:
Yeah ... I could've gone either way (on the class/loose function). I just wanted to be able to use it from `lib/instance'.

Reply:
I just realized I forgot to use it there. (rolling eyes to myself)

Reply:
I'll add a comment on the method

Reply:
I'd be OK with this being a static function on instance: `instance.SerializeIsVisibleFn = function (options = []) {`.  Could also put it as a static on `lib/instance.js` or make a `lib/resources/util.js` and have it wild there.  But yeah, I agree that this doesn't make sense as a class function, so any of those alternates are cool with me.

Reply:
Static it is! :-)

Reply:
Actually ... I think it would be better in `lib/resources/util.js`